packages/webdriver/src/request.ts

```diff
@@ -23,24 +35,31 @@ const agents = {
 }
 
 export default class WebDriverRequest extends EventEmitter {
-    constructor (method, endpoint, body, isHubCommand) {
+    body: object
```

Comment:
I think we should avoid using `object` as a type and replace it with `Record<string, unknown>` or something.

See also https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/ban-types.md

Reply:
let's put such eslint rule in place with a separate PR, will fix this.